.github/workflows/test-contracts.yml

```diff
@@ -19,7 +19,7 @@ concurrency:
 jobs:
   forge-tests:
     name: Foundry Project
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
This change is needed because of a GLIBC dependency issue

Reply:
PR with this change has been merged to master already